_posts/2020-10-23-kubernetes-webhooks-made-easy-with-openfaas.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Kubernetes Webhooks made easy with OpenFaaS"
+description: "Extending the Kubernetes with Admission webhook using OpenFaaS Serverless Functions"
+date: 2020-10-23
+image: /images/2020-10-github-actions/lego.jpg
```

Comment:
Please pick your own background for the post and resize as per the README - it gives dimensions

Unplash and pexels are both good websites for royalty free stock photography.

Reply:
done.